javascript-loops-practice/filter-out-strings.js

```diff
@@ -0,0 +1,13 @@
+/* exported filterOutStrings */
+function filterOutStrings(values) {
+
+  var newArr = [];
+  for (var i = 0; i < values.length; i++) {
+    if (typeof values[i] === 'string') {
+      delete (values[i]);
```

Comment:
Nooooooo, this will change the original array's value 😭

If it is a string, just ignore it, don't delete it.